Readme.md

```diff
@@ -62,14 +62,20 @@ sizeOf('images/funny-cats.png')
 
 ### Async/Await (Typescript & ES7)
 ```javascript
-var { promisify } = require('util');
-var sizeOf = promisify(require('image-size'));
-try {
-  const dimensions = await sizeOf('images/funny-cats.png');
-  console.log(dimensions.width, dimensions.height);
-} catch (err) {
-  console.error(err);
+const { promisify } = require('util');
+const sizeOf = promisify(require('image-size'));
+
+const getDimensions = async () => {
+    try {
+        const dimensions = await sizeOf('images/funny-cats.png');
+        return dimensions;
+    } catch (err) {
+        console.error(err);
+    }
 }
+
+getDimensions().then(c => console.log(c));
```

Comment:
Sure the ['top level' await](https://github.com/tc39/proposal-top-level-await) is not yet supported...

One of the goals of async / await is to not need `.then()` & callbacks anymore

I would then have kept the `console.log()` in the try block

You may, if you prefer, also rename `getDimension()` to `showDimensionor()` or `outputDimension()`